packages/vscode-pyright/src/extension.ts

```diff
@@ -260,7 +260,7 @@ export async function activate(context: ExtensionContext) {
             },
         },
         initializationOptions: {
-            diagnosticMode: workspace.getConfiguration('python.analysis').get('diagnosticMode'),
+            diagnosticMode: workspace.getConfiguration('basedpyright.analysis').get('diagnosticMode'),
             disablePullDiagnostics: workspace.getConfiguration('pyright').get('disablePullDiagnostics'),
```

Comment:
this should probably be updated too for consistency
```suggestion
            disablePullDiagnostics: workspace.getConfiguration('basedpyright').get('disablePullDiagnostics'),
```
also needs to be updated in `packages/vscode-pyright/package.json`

Reply:
Indeed, I've pushed a fix 👍🏻